### DISCUSSION TITLE
Values comparison fails after 2.3.0

### DISCUSSION THREAD

**Discussion by pando85** [mitsuhiko/minijinja#582] [Q&A] [answered]
Hello,

I'm using minijinja since 2.0.0 and it is wonderful. But in the last update my program tests where broken because something like this:
```
    #[test]
    fn test_minijinja() {
        let vars = context! {};
        let new_vars = context! {..vars.clone()};
        assert_eq!(vars, new_vars);
    }
```
Anyone could guide me about why in minijinja 2.2.0 this worked and now I get an error? Is this an intended behavior? Any workarounds?

Thanks!

**Comment by mitsuhiko** [comment, marked as answer]
Filed as bug in #583

**Reply by pando85** [reply]
That was fast. Thanks a lot.
